scripts/modules/MsIndustryLinks/README.md

```diff
@@ -22,7 +22,7 @@ The **MsIndustryLinks** module contains cmdlets to generate workflow templates (
 | [New-CustomConnector](customConnector/New-CustomConnector.md)                              | Create a custom connector in a Power Platform environment                             |
 | [New-CustomConnectorConfig](customConnector/New-CustomConnectorConfig.md)                  | Creates Power Platform custom connector asset configuration files                     |
 | [New-DataSourceWorkflow](templates/data_source/New-DataSourceWorkflow.md)                  | Generate an end-to-end workflow template for specified data source                    |
-| [New-IngestionWorkflow](templates/data_sink/New-IngestionWorkflow.md)                      | Generate an ingestion workflow template                                               |
+| [New-DataSinkWorkflow](templates/data_sink/New-DataSinkWorkflow.md)                        | Generate an ingestion workflow template                                               |
```

Comment:
Suggestion to update the description to contain "data sink" instead of ingestion

Reply:
Whoops, missed that. Will do.